src/test/java/api/loans/anonymization/AnonymizeLoansAfterXIntervalTests.java

```diff
@@ -432,4 +434,19 @@ public void testClosedLoanWithOpenFeesAndFinesNotAnonymizedIntervalPassed()
     assertThat(loansStorageClient.getById(loanID)
         .getJson(), not(isAnonymized()));
   }
+
+  @Test
+  public void testNoConfigurationExceptionIsHandled()
+    throws InterruptedException, ExecutionException, TimeoutException, MalformedURLException {
+
+    IndividualResource loanResource = loansFixture.checkOutByBarcode(new CheckOutByBarcodeRequestBuilder().forItem(item1)
```

Comment:
This needs to be split into lines that meet the maximum line length expectations